web/frontend/src/components/navigation/index.svelte

```diff
@@ -6,14 +6,15 @@ import 'maplibre-gl/dist/maplibre-gl.css';
 import { notify } from '@viamrobotics/prime';
 import { navigationApi, NavigationClient, type ServiceError } from '@viamrobotics/sdk';
 import { getObstacles, type NavigationModes } from '@/api/navigation';
-import { mapCenter, centerMap, robotPosition, flyToMap, write as writeStore, obstacles } from './stores';
+import { mapCenter, centerMap, robotPosition, flyToMap, write as writeStore, obstacles, navMode } from './stores';
```

Comment:
Since we're only using `navMode` in this file we can do away with stores, in svelte variables themselves trigger reactive changes in html if they're updated, so something like this should work:

```svelte
<script lang='ts'>
  let navMode: 'Manual' | 'Waypoint' | '' = ''
  
  // ...later
  navMode = 'Manual'
<script>

<v-radio
  label="Navigation mode"
  options="Manual, Waypoint"
  selected={navMode}
  on:input={setNavigationMode}
/>